lib/charCounter.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+/* eslint-disable */
+
+/**
+ * Function to count how many characters are remaining in text fields
+ *
+ * @param {Object} input - inputs in this plugin's instance and their values as {String}
+ * *
+ * @module charCounter
+ */
+
+module.exports = function charCounter(input) {
+  var msg = input.parent.querySelector('.input-plugin-textarea'),
+      charRemain = input.parent.querySelector('.feedback-number'),
+      maxChar = input.parent.querySelector('.input-plugin-textarea').maxLength,
```

Comment:
Settings should be passed in. Don't fake it with custom data attribtues.

Also, if you were going to do this, the actual attribute you've created in the HTML isn't camel case, and it should be `msg.getAttribute('maxlength')`